src/System.Windows.Forms/src/System/Windows/Forms/CheckBox.CheckBoxAccessibleObject.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+using static System.Windows.Forms.Control;
+using static Interop;
+
+namespace System.Windows.Forms
+{
+    public partial class CheckBox
+    {
+        [ComVisible(true)]
+        internal class CheckBoxAccessibleObject : ControlAccessibleObject
+        {
+            internal CheckBoxAccessibleObject(CheckBox owner) : base(owner)
+            {
+            }
```

Comment:
Please add check owner for null.

Reply:
Reviewed and found out that the base has already this check in the constructor (was added recently). Sorry for confusion.